src/components/menu-bar/menu-bar.jsx

```diff
@@ -791,6 +789,8 @@ const mapStateToProps = state => {
         projectTitle: state.scratchGui.projectTitle,
         sessionExists: state.session && typeof state.session.session !== 'undefined',
         username: user ? user.username : null,
+        userOwnsProject: ownProps.authorUsername && user &&
+            (ownProps.authorUsername === user.username),
```

Comment:
I believe we discussed creating this prop -- it makes for cleaner separation of props logic and component display logic.